src/leaguepedia_client.py

```
@@ -93,13 +109,186 @@ async def fetch_upcoming_matches(self, tournament_name_like: str):
+                        ):
+                            retry_after = None
+                            if "Retry-After" in getattr(resp, "headers", {}):
+                                try:
+                                    retry_after = float(
+                                        resp.headers["Retry-After"]
+                                    )
+                                except (ValueError, TypeError):
+                                    pass
+                            return True, retry_after
+            except Exception:
+                pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)